targets/targets.json

```diff
@@ -4008,5 +4008,60 @@
         "detect_code": ["7013"],
         "release_versions": ["5"],
         "bootloader_supported": true
+    },
+    "ARM_FM_Target": {
+        "inherits": ["Target"],
+        "public": false,
+        "device_has": ["AACI", "ANALOGIN", "CLCD", "ETHERNET", "I2C", "INTERRUPTIN", "PORTIN", "PORTINOUT", "PORTOUT", "SERIAL", "SERIAL_FC", "SPI", "SPISLAVE", "TSC"]
+    },
+    "FVP_MPS2_M0": {
+        "inherits": ["ARM_FM_Target"],
+        "core": "Cortex-M0",
+        "supported_toolchains": ["GCC_ARM", "ARM", "IAR"],
+        "extra_labels": ["ARM_FM", "FVP_MPS2", "FVP_MPS2_M0"],
```

Comment:
Move "ARM_FM" (should be the parent name) and `"FVP_MPS2"` to the parent, and change this to an `extra_labels_add`